index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div>
+        <p class="header-microverse">Hello Microverse!</p>
```

Comment:
As per the requirement of this project: 
![image](https://user-images.githubusercontent.com/43172164/196746697-af3823fa-edd7-4b7e-9e4d-40e2ea99d62e.png)
- In your code, you are using a `p` tag but you should be using a `h1` tag.